src/types/ioredis-cluster-commands.d.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * AR.IO Gateway
+ * Copyright (C) 2022-2025 Permanent Data Solutions, Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+declare module 'ioredis' {
```

Comment:
mmm - does installing `yarn add @types/ioredis` remove the need for this file?

Reply:
Nope. We're defining custom commands.